test/unit/app/prices/get-current-price.spec.ts

```diff
@@ -1,25 +1,30 @@
-import { Prices } from "@app"
-
 import { CacheKeys } from "@domain/cache"
 import { DisplayCurrency } from "@domain/fiat"
 import { PriceNotAvailableError } from "@domain/price"
 
 import * as PriceServiceImpl from "@services/price"
-import { LocalCacheService } from "@services/cache"
-
-jest.mock("@services/redis", () => ({}))
+import { LocalCacheService } from "@services/cache/local"
+import { getCurrentSatPrice, getCurrentUsdCentPrice } from "@app/prices"
```

Comment:
this is not a good change... I tested locally and the only required change is `import { LocalCacheService } from "@services/cache/local"`